### DISCUSSION TITLE
Ignore some file types or some way to suppress "unsupported file extensions" warnings?

### DISCUSSION THREAD

**Discussion by wonrax** [contentlayerdev/contentlayer#459] [Q&A]
I want to put images in the same directory with the MDX post like this:
```
content/
├─ blog/
│  ├─ post-01/
│  │  ├─ index.mdx
│  │  ├─ image.png
```
```ts
export const BlogPost = defineDocumentType(() => ({
  name: "BlogPost",
  filePathPattern: `blog/**/*.mdx`,
  contentType: "mdx",
  ...
}
```

But contentlayer keeps logging warning for every single file. Couldn't imagine if I have 1000 images.
```shell
next-www:dev: Warning: Found 1 problems in 1 documents.
next-www:dev:  └── Found unsupported file extensions for 1 documents. (Skipping documents)
next-www:dev:      • "blog/post-01/image.png" uses "png"
```

Is there anyway to suppress these warnings? Or an option to ignore certain file extensions in `defineDocumentType`?

**Comment by schickling** [comment]
Have you tried using the `contentDirExclude` option to exclude the `.png` files?
